testgrid/config.yaml

```diff
@@ -7804,6 +7813,15 @@ dashboards:
   - name: pr-k8s-e2e
     description: Run e2e tests
     test_group_name: pull-kubernetes-e2e-aks-engine-azure
+  - name: pr-cloud-provider-e2e
+    description: Run Kubernetes e2e tests for cloud-provider-azure
```

Comment:
Should we also include `pull-cloud-provider-azure-e2e-ccm` here?

Reply:
Yep, doing it in another PR #11846.

Reply:
But I missed it, but I do not see the `pull-cloud-provider-azure-e2e-ccm` job under `- name: sig-azure-master` in testgrid/config.yaml https://github.com/kubernetes/test-infra/pull/11846/files#diff-0e9f7582435fdc364e5f7adce048d956R7805

Reply:
Oh, missed it. Let me add it back.